OpenRA.Mods.Common/Traits/Mobile.cs

```diff
@@ -221,7 +221,7 @@ void INotifyRemovedFromWorld.RemovedFromWorld(Actor self)
 
 		public void Nudge(Actor self, Actor nudger, bool force)
 		{
-			if (IsTraitDisabled)
+			if (IsTraitDisabled || IsTraitPaused)
```

Comment:
`MobileProperties.IsMobile` and `VisualMoveIntoTarget.Tick` will probably need to be updated in the same way.

Reply:
Done.